config/initializers/kaminari_config.rb

```diff
@@ -0,0 +1,10 @@
+Kaminari.configure do |config|
```

Comment:
We're just using the defaults for now so there's no need for this file, we can always add it back in later if we do want to tweak some settings.